csrc/torch_memory_saver.cpp

```diff
@@ -166,27 +180,41 @@ class TorchMemorySaver {
 
 #ifdef TMS_DEBUG_LOG
         std::cout << "[torch_memory_saver.cpp] TorchMemorySaver.cuda_free "
-                  << " ptr=" << ptr << " metadata.size=" << metadata.size
-                  << " metadata.allocHandle=" << metadata.allocHandle << " tag=" << metadata.tag
+                  << " ptr=" << ptr << " metadata.size=" << metadata.size << " metadata.allocHandle="
+                  << metadata.allocHandle << " tag=" << metadata.tag
                   << std::endl;
 #endif
 
         return cudaSuccess;
     }
 
-    void pause(const std::string& tag) {
+    int pause(const std::string& tag) {
```

Comment:
nit: maybe we just directly print+exit(1) on failures to simplify code, since users should never do that (unless the code is buggy) and it seems OK to exit on that case

Reply:
yes, but I do not expect python to catch it - this is a bug and programmers should fix it in their code and rerun the program and see no error